### PR TITLE
[CI][VSTS] Add a poor mans debugger.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: tools/devops/automation/templates/build/build.yml [hunk 1]
```diff
@@ -136,82 +138,87 @@ steps:
     # a branch in origin
     
     if ($buildReason -eq "PullRequest" -or (($buildReason -eq "Manual") -and ($buildSourceBranchName -eq "merge")) ) {
-      Write-Host '##vso[task.setvariable variable=IsPR;isOutput=true]False'
+      $configVars.Add("IsPR", "True")
```
Comment (mandel-macaque) [inline tools/devops/automation/templates/build/build.yml:141]: The change of the value is literally the reason I created this PR, This is a PR and was set to false, probably done in a bad merge.

**Review by spouliot — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Test failures are known

**Pull request merged**